Makefile

```diff
@@ -0,0 +1,2 @@
+flake: 
+	flake8 --exclude migrations powerdns/
```

Comment:
my bad. powerdns it the directory to run flake for (first i thought it is excluded too). So another question - don't we want to check flakes in dnsaas too? :)